src/core/features/notes/controller/NotesController.test.ts

```diff
@@ -96,6 +96,14 @@ describe('CRUD operations', () => {
 
 		await db.close();
 	});
+
+	test('delete with empty list does not throw', async () => {
+		const dbFile = createFileControllerMock();
+		const db = await openDatabase(dbFile);
+		const registry = new NotesController(db, FAKE_WORKSPACE_ID);
+
+		await expect(registry.delete([])).resolves.toBeUndefined();
```

Comment:
nit: use `.not.toThrow()` if your intention is to check that call does not throw